packages/angular/src/table/head/table-head.component.ts

```diff
@@ -43,8 +43,13 @@ import { TableHead } from 'carbon-components-angular';
             [attr.rowspan]="column.rowSpan"
             [filterTitle]="getFilterTitle()"
             (sort)="sort.emit(i)"
-            [class]="column.className"
+            [class]="column.className + ' iot--table-head--table-header'"
```

Comment:
would one of these make more sense in ngClass?

Reply:
not sure, neither are conditional